pkg/proxy/plugin_test.go

```diff
@@ -0,0 +1,121 @@
+package proxy
+
+import (
+	"context"
+	"errors"
+	"github.com/lni/goutils/leaktest"
+	"github.com/matrixorigin/matrixone/pkg/common/runtime"
+	"github.com/matrixorigin/matrixone/pkg/frontend"
+	"github.com/matrixorigin/matrixone/pkg/pb/metadata"
+	"github.com/matrixorigin/matrixone/pkg/pb/plugin"
+	"github.com/stretchr/testify/require"
+	"testing"
```

Comment:
please seperate internal packages and external packages with a blank line, and add copyright header.